### ISSUE TITLE
browserify(commonJS) support

### ISSUE THREAD

**Issue by jpablom** [bgrins/spectrum#373]
colorpicker doesn't work with commonJS environment.
If use 

```javascript
require('spectrum-colorpicker');

$(el).spectrum({
    showAlpha: true
    // some other options
});
```

throws `$(...).spectrum is not a function`

in the commonJS part of umd declaration, `jquery` is missed as dependency
```javascript
    if (typeof define === 'function' && define.amd) { // AMD
        define(['jquery'], factory);
    }
    else if (typeof exports == "object" && typeof module == "object") { // CommonJS
        module.exports = factory;
    }
    else { // Browser
        factory(jQuery);
    }
```

solved with
```javascript
 if (typeof define === 'function' && define.amd) { // AMD
        define(['jquery'], factory);
    }
    else if (typeof exports == "object" && typeof module == "object") { // CommonJS
        module.exports = factory(require('jquery'));
    }
    else { // Browser
        factory(jQuery);
    }
```

**Comment by furey** [comment]
I've just hit this issue.

Is this a simple fix in the master branch?

Do you require a pull request?

Cheers!

**Comment by poohitan** [comment]
Hit the same issue. Would it be fixed soon?

**Comment by jpablom** [comment]
i made pull request #379 for this

**Comment by bgrins** [comment]
This should be fixed now by https://github.com/bgrins/spectrum/commit/d60bde7b7d875be4d9c7ad62d95af3e5e26462bc.  Please let me know if it isn't.

**Comment by jpablom** [comment]
yeah, as i said in the description of the issue adding `module.exports = factory(require('jquery'))` works

**Comment by bgrins** [comment]
Thanks for the report and fix.  I'll tag a release with the update

**Comment by jpablom** [comment]
thanks @bgrins

**Comment by jpablom** [comment]
I appreciate it if you will update the npm registry

**Comment by bgrins** [comment]
published 1.8.0: https://www.npmjs.com/package/spectrum-colorpicker
